content/docs/guides/troubleshooting/traffic_management/iptables_redirection.md

```diff
@@ -180,3 +180,96 @@ Init Containers:
 In the example above, the following `iptables` commands are responsible for explicitly ignoring the configured outbound ports (`6379, 7070 and 8080`) from being redirected to the Envoy proxy sidecar.
 ```console
 iptables -t nat -I PROXY_OUTPUT -p tcp --match multiport --dports 6379,7070,8080 -j RETURN
+```
+
+## When network interface exclusions are configured
+
+By default, all traffic using TCP as the underlying transport protocol are redirected via the Envoy proxy sidecar container. This means all TCP based outbound traffic from applications are redirected and routed via the Envoy proxy sidecar based on service mesh policies. When network interface exclusions are configured, traffic belonging to these network interfaces will not be proxied to the Envoy sidecar.
+
+If network interfaces are configured to be excluded but being subject to service mesh policies, verify they are configured as expected.
```

Comment:
This wording is a little confusing. Perhaps reword to something like "If you have network interfaces that are subjected to mesh policies, verify their configuration to ensure they are properly excluded." Or something like that.

Reply:
Fixed